packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -175,6 +180,11 @@ export class FrontendPluginImpl implements PluginImpl {
   }
 
   public async localDebug(ctx: PluginContext): Promise<TeamsFxResult> {
+    if (isConfigUnifyEnabled()) {
+      ctx.envInfo.state
+        .get(PluginInfo.PluginName)
+        ?.set(FRONTEND_INDEX_PATH, Constants.FrontendIndexPath);
+    }
     ctx.localSettings?.frontend?.set(
```

Comment:
Do we still need to set localSettings when configUnifyEnabled?

Reply:
No, after config has been fully updated ```localSettings``` will be removed.